test/convert_test.dart

```diff
@@ -32,15 +32,15 @@ main() {
       test('proxy array', () {
         var model = new MyModel();
         var list = [1, model, 'a'];
-        var array = jsValue(list) as JsArray;
-        expect(dartValue(array), list);
+        var array = convertToJs(list) as JsArray;
+        expect(convertToDart(array), list);
       });
 
       test('object', () {
         var model = new MyModel();
         var object =
-            new JsObject.jsify({'1': 1, 'model': jsValue(model), 'a': 'a',});
-        var dartMap = dartValue(object) as Map;
+            new JsObject.jsify({'1': 1, 'model': convertToJs(model), 'a': 'a',});
```

Comment:
I can't tell in github PRs whether the lines got too long, so if you haven't you may want to run dartfmt